cfgov/prepaid_agreements/jinja2/prepaid_agreements/index.html

```diff
@@ -0,0 +1,100 @@
+            <div class="lead-paragraph u-readable-width">
+                Find prepaid product agreements from more than 80 issuers. You can search agreements by issuer name, product name, prepaid account type, and other critera.
+            </div>
+            <p>Note that we are unable to do a full-text search of agreement files.</p>
+            <p>You may also choose to download all prepaid agreements as a <a href="https://files.consumerfinance.gov/a/assets/prepaid-agreements/bulk.zip">single compressed file</a> (1.3 GB, updated nightly).
+        </div>
+
+        <div class="search_wrapper u-mt30">
+            <form class="u-mt15" method="get" action="." >
+                <div id="search-hidden-filters">
+                {% for filter in filters %}
+                    {% if filter in valid_filters %}
+                        {% for val in filters[filter] %}
+                            <input type="hidden" name="{{ filter }}" value="{{ val }}">
+                        {% endfor %}
+                    {% endif %}
+                {% endfor %}
+                </div>
+
+                <h3 class="h4 u-mb5">Search within</h3>
+                <div class="layout-row u-mb30">
+                    <div class="a-select flex-fixed">
+                        {% set opts = [
+                            ('All fields', 'all'),
+                            ('Product name', 'name'),
+                            ('Program manager', 'program_manager'),
+                            ('Other relevant party', 'other_relevant_parties')
+                        ]%}
+                        <select aria-label="Choose which field will be searched"
+                                id="search_field"
+                                name="search_field">
+                            <optgroup label="Search within">
+                                {% for label, val in opts %}
+                                <option value="{{ val }}"
+                                        {{ 'selected' if search_field == val else '' }}>
+                                    {{ label }}
+                                </option>
+                                {% endfor %}
+                            </optgroup>
+                        </select>
+                    </div>
+                    <div class="flex-all">
+                        <div class="m-btn-inside-input input-contains-label">
+                            <label for="searchText" class="input-contains-label_before input-contains-label_before__search">
+                                {{ svg_icon('search') }}
+                                <span class="u-visually-hidden">The term to search for</span>
+                            </label>
+                            <input id="searchText" 
+                                   type="text"
+                                   autocomplete="off"
+                                   class="a-text-input"
+                                   name="q"
```

Comment:
Ensure `q` doesn't collide with another search field on the page.

Reply:
`q` is the standard field name we use for in-page search -- we use `query` for the main search in the header.